tests/ui/ChatterOverlayPlacement.test.jsx

```
@@ -0,0 +1,109 @@
+ */
+const stubRects = ({ overlay, obstacle }) => {
+  Element.prototype.getBoundingClientRect = function () {
+    if (this.getAttribute('role') === 'status') return overlay
+    if (this.hasAttribute('data-chatter-avoid')) return obstacle
+    return { left: 0, top: 0, right: 0, bottom: 0, width: 0, height: 0 }
+  }
+}
+
+afterEach(() => {
+  Element.prototype.getBoundingClientRect = realGetRect
+  document.querySelectorAll('[data-chatter-avoid]').forEach(el => el.remove())
```

Biome (2.5.7)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)